website/source/docs/job-specification/reschedule.html.md

```diff
@@ -4,9 +4,9 @@ page_title: "reschedule Stanza - Job Specification"
 sidebar_current: "docs-job-specification-reschedule"
 description: |-
   The "reschedule" stanza specifies the group's rescheduling strategy upon
-  allocation failures. The reschedule strategy can be configured with number
-  of attempts and a time interval. Nomad will only attempt to reschedule
-  failed allocations on to another node only after any local [restarts](docs/job-specification/restart.html)
+  allocation failures. The reschedule strategy can be configured with options
+  described below. Nomad will only attempt to reschedule failed allocations on
```

Comment:
The described below doesn't mean much since this text won't be displayed above the options. Could probably delete the sentence